src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {

```

Comment:
Static method is forbidden by task description
```suggestion
    public Ball getRandomBall() {

```